.github/workflows/diff.yml

```diff
@@ -14,15 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     container: obolibrary/odklite:v1.3.0
     steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: '#gogoeditdiff'
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
       - uses: actions/checkout@v3
+        if: steps.check.outputs.triggered == 'true'
       # Checks-out main branch under "main" directory
       - uses: actions/checkout@v3
+        if: steps.check.outputs.triggered == 'true'
         with:
           ref: master
           path: master
       - name: Diff classification
+        if: steps.check.outputs.triggered == 'true'
         run: export ROBOT_JAVA_ARGS=-Xmx6G; robot diff --labels True --left master/src/ontology/cl-edit.owl --left-catalog master/src/ontology/catalog-v001.xml --right src/ontology/cl-edit.owl --right-catalog src/ontology/catalog-v001.xml -f markdown -o edit-diff.md
```

Comment:
I think you can do make steps for PR and master in here rather than other steps. And also you can add the other robot diff step in this one. You are downloading the master and current branch multiple times in the action. I don't think we need that. @gouttegd and @matentzn can comment on this one as well.

Reply:
Like we did in the Uberon diff action, https://github.com/obophenotype/uberon/blob/bd56184537d6503487d94c9103814a9876781acb/.github/workflows/diff.yml#L31-L47

Reply:
Oh I think the seperation of master and branch make step is intentional when we copied it over from @balhoff - this allows for simultaneous runs rather than waiting for one to finish to start the next:
![Screenshot 2022-11-17 at 15 17 44](https://user-images.githubusercontent.com/76212760/202485397-f0c809e0-f2cd-45ff-a02f-ba605b6b29d3.png)